lib/compat/wordpress-6.2/script-loader.php

```diff
@@ -64,89 +64,10 @@ static function () use ( $style ) {
 	);
 }
 
-/**
- * Sets the content assets for the block editor.
- *
- * Note for core merge: see inline comment on what's been updated.
- */
-function gutenberg_resolve_assets_override() {
-	global $pagenow;
-
-	$script_handles = array(
-		'wp-polyfill',
-	);
-	// Note for core merge: only 'wp-edit-blocks' should be in this array.
-	$style_handles = array(
-		'wp-edit-blocks',
-	);
-
-	if ( current_theme_supports( 'wp-block-styles' ) ) {
-		$style_handles[] = 'wp-block-library-theme';
-	}
-
-	if ( 'widgets.php' === $pagenow || 'customize.php' === $pagenow ) {
```

Comment:
@ellatrix Is there any part of this that we should have at `_gutenberg_get_iframed_editor_assets`? This bit for the widgets screen caught my eye.

Reply:
We should check that in #50091, but as far as I could tell, these editors were not iframed.